src/NUnitFramework/tests/Constraints/EqualConstraintTests.cs

```diff
@@ -666,6 +666,66 @@ public void SameValueDifferentTypeExactMessageMatch()
             Assert.AreEqual(ex.Message, "  Expected: 0 (Int32)"+ NL + "  But was:  0 (IntPtr)"+ NL);
         }
 
+        class Dummy
+        {
+            internal readonly int value;
+
+            public Dummy(int value)
+            {
+                this.value = value;
+            }
+
+            public override string ToString()
+            {
+                return "Dummy " + value;
+            }
+        }
+
+        class Dummy1
+        {
+            internal readonly int value;
+
+            public Dummy1(int value)
+            {
+                this.value = value;
+            }
+
+            public override string ToString()
+            {
+                return "Dummy " + value;
+            }
+        }
+
+        class DummyTemplatedClass<T>
```

Comment:
Complete nit: the IL/C# lingo is "generic"– C++ has templates, IL and C# do not.

Reply:
Whoops.  Will rename then.